static/absolute/js/searchPage.js

```diff
@@ -90,6 +89,14 @@ window.onload = function () {
   });
 };
 
+async function initInviteButton() {
+  const response = await (await fetch(`api/${getEventId()}/name`)).json();
+  console.log(response.data);
```

Comment:
I am suspicious of generic logging like this, is this necessary? If so, let's try to make clear what and why it's being logged.

Reply:
Whoops - this was a stray `console.log`